src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -83,10 +88,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
     /**
      * Loads the Event Dispatcher from the container.
      *
+     * @param string $serviceId
+     *
      * @return EventDispatcherInterface
      */
-    protected function getEventDispatcher()
+    protected function getEventDispatcher($serviceId)
```

Comment:
Yep, I wonder why. 
To avoid this BC break I can store the serviceId as a private property of the command and use it directly in this method.